src/components/Match/MatchStory.jsx

```diff
@@ -540,6 +554,14 @@ const generateStory = (match) => {
   // Intro
   events.push(new IntroEvent(match));
 
+  // Prediction
+  match.players.forEach((player) => {
+    if (player.pred_vict === true) {
+      const order = player.player_slot < 5 ? player.player_slot - 89 : player.player_slot - 212;
```

Comment:
StoryEvents use the time of an event as their unique key, but predictions don't have a time stamp

So to get a unique key per instance of PredictionEvent I normalized player_slot to the -89-80 range using 89 for radiant (0-4) slots and 212 for dire (127-132) slots